addons/confirm-actions/addon.json

```diff
@@ -53,11 +53,17 @@
       "id": "closingtopic",
       "type": "boolean",
       "default": false
+    },
+    {
+      "name": "Confirm removing projects",
+      "id": "removingprojects",
+      "type": "boolean",
+      "default": false
     }
```

Comment:
Rename this to "Confirm deleting projects" and move it after "Confirm project unsharing".

I think it should be enabled by default.